modules/user_accounts/templates/menu_user_accounts.tpl

```diff
@@ -2,7 +2,7 @@
 
 <!-- start the selection table -->
 <div class="row">
-    <div class="col-sm-9 col-lg-8">
+    <div class="col-sm-9 col-lg-12">
```

Comment:
I think I know what you're talking about but I don't think it exists in `user_accounts`. Definitely lies somewhere else though.